locale/en_GB/LC_MESSAGES/djangojs.po

```diff
@@ -70,46 +70,34 @@ msgstr ""
 "will be deleted if it does not meet our discussion rules (netiquette)."
 
 #: adhocracy4/comments/static/comments/Comment.jsx:96
-#: adhocracy4/comments_async/static/comments_async/comment.jsx:296
```

Comment:
Seems like alot of js strings are getting removed again, however, they still seem to be in the code. I still don't really get why `make po` is so unreliable

Reply:
oh, i didn't doublecheck them tbh. Do you know what i can do about it?

Reply:
no, sadly I have no idea. Last time @phillimorland just ran make po again and pushed it

Reply:
maybe removing those strings is also correct and I just don't get it :sweat_smile:

Reply:
Yeah, that is the problem why we always put the strings on top in the jsx-Code. And then sometimes they are still gone. So, if there weren't changes and a lot of strings are lost you just have to check if they are still in the code and try `make po` again or ask someone else to do it or try to put the strings to the top. It's pretty annoying.